src/examples/createSignatureOrder/basic.cs

```diff
@@ -0,0 +1,25 @@
+using Criipto.Signatures;
```

Comment:
Maybe a sample-specific namespace here (same comment for all .cs files) will fix the compilation issue.

Reply:
@sgryt That won't work, the file has top level code which isn't allowed with a namespace

Reply:
I'm confused now: Aren't we trying to get rid of top-level code (to fix the compilation error) ??